### PR TITLE
Removes a lot of traitor items, adds a few

### PR DESCRIPTION
**Pull request by Muncher2112**
author: Muncher21
changes:

    rscadd: "Removed lots traitor items.  Either useless, didn't fit the lore, or too powerful"
    rscadd: "Added a few new items.  Unrevelator syringe, switch-blade, throwing stars, bear trap"

### CODE REVIEW

File: code/datums/uplink/stealth_and_camouflage_items.dm [hunk 1]
```diff
@@ -14,6 +14,12 @@
 	item_cost = 8
 	path = /obj/item/weapon/storage/box/syndie_kit/spy
 
+/datum/uplink_item/item/stealth_items/voice
```
Comment (mattroks101) [inline code/datums/uplink/stealth_and_camouflage_items.dm]: This no longer works.

Comment (mattroks101) [reply code/datums/uplink/stealth_and_camouflage_items.dm]: The mask I mean. It should be removed from traitor uplink.

Comment (Muncher2112) [reply code/datums/uplink/stealth_and_camouflage_items.dm]: It's out now

File: code/modules/paperwork/pen.dm [hunk 2]
```diff
@@ -115,6 +115,13 @@
 	reagents.add_reagent("zombiepowder", 10)
 	reagents.add_reagent("cryptobiolin", 15)
 
+/obj/item/weapon/pen/reagent/unrevelator
+	desc = "A syringe full of unrevelator.  Great for avoid pesky zealots trying to scan you."
```
Comment (mattroks101) [inline code/modules/paperwork/pen.dm]: *avoiding

**Pull request merged**